<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cell_userInformation</class>
 <widget class="QDialog" name="cell_userInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户信息</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QFrame" name="frame_photo">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>头像</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>19</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" rowspan="3">
          <widget class="QFrame" name="frame_head">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_image">
              <property name="minimumSize">
               <size>
                <width>141</width>
                <height>121</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>112</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pushButton_reSelect">
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #4CAF50; /* 设置按钮背景颜色 */
    color: white; /* 设置文字颜色 */
    border: 1px solid #4CAF50; /* 设置按钮边框 */
    border-radius: 5px; /* 设置按钮圆角 */
    padding: 5px 10px; /* 设置按钮内边距 */
    font-size: 14px; /* 设置文字大小 */
}

QPushButton:hover {
    background-color: #45a049; /* 鼠标悬停时的背景颜色 */
}

QPushButton:pressed {
    background-color: #3e8e41; /* 鼠标按下时的背景颜色 */
}
</string>
           </property>
           <property name="text">
            <string>重新选择</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>112</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="2">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>112</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="pushButton_savePhoto">
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #4CAF50; /* 设置按钮背景颜色 */
    color: white; /* 设置文字颜色 */
    border: 1px solid #4CAF50; /* 设置按钮边框 */
    border-radius: 5px; /* 设置按钮圆角 */
    padding: 5px 10px; /* 设置按钮内边距 */
    font-size: 14px; /* 设置文字大小 */
}

QPushButton:hover {
    background-color: #45a049; /* 鼠标悬停时的背景颜色 */
}

QPushButton:pressed {
    background-color: #3e8e41; /* 鼠标按下时的背景颜色 */
}
</string>
           </property>
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>112</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="2">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>19</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame_other">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>昵称</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_user"/>
           </item>
          </layout>
         </item>
         <item row="0" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>性别</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_gender">
             <property name="minimumSize">
              <size>
               <width>236</width>
               <height>0</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>男</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>女</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>保密</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>生日</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEdit_birthday">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>地区</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_city"/>
           </item>
          </layout>
         </item>
         <item row="2" column="0" colspan="3">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>邮箱</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_mail">
             <property name="minimumSize">
              <size>
               <width>430</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>个人说明</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QPlainTextEdit" name="plainTextEdit_description"/>
         </item>
         <item row="4" column="1" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="pushButton_save">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #4CAF50; /* 设置按钮背景颜色 */
    color: white; /* 设置文字颜色 */
    border: 1px solid #4CAF50; /* 设置按钮边框 */
    border-radius: 5px; /* 设置按钮圆角 */
    padding: 5px 10px; /* 设置按钮内边距 */
    font-size: 14px; /* 设置文字大小 */
}

QPushButton:hover {
    background-color: #45a049; /* 鼠标悬停时的背景颜色 */
}

QPushButton:pressed {
    background-color: #3e8e41; /* 鼠标按下时的背景颜色 */
}
</string>
             </property>
             <property name="text">
              <string>保存</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_quit">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #4CAF50; /* 设置按钮背景颜色 */
    color: white; /* 设置文字颜色 */
    border: 1px solid #4CAF50; /* 设置按钮边框 */
    border-radius: 5px; /* 设置按钮圆角 */
    padding: 5px 10px; /* 设置按钮内边距 */
    font-size: 14px; /* 设置文字大小 */
}

QPushButton:hover {
    background-color: #45a049; /* 鼠标悬停时的背景颜色 */
}

QPushButton:pressed {
    background-color: #3e8e41; /* 鼠标按下时的背景颜色 */
}
</string>
             </property>
             <property name="text">
              <string>退出</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
